rmgpy/data/thermo.py

```diff
@@ -1487,6 +1487,11 @@ def get_thermo_data_for_surface_species(self, species):
                 bond.increment_order()
                 adsorbed_atoms[0].decrement_radical()
                 adsorbed_atoms[1].decrement_radical()
+                if adsorbed_atoms[0].radical_electrons and adsorbed_atoms[1].radical_electrons:
+                    # There are still spare adjacenct radicals, so do it again
+                    bond.increment_order()
+                    adsorbed_atoms[0].decrement_radical()
+                    adsorbed_atoms[1].decrement_radical()
```

Comment:
I am assuming that something like X#R-R#X would never happen, right? If so the atoms would have lone pairs instead of radicals. Though even then I guess they would keep their lone pairs instead of forming another bond? I am sure this is fine, it is just something that I was curious about.